PHP/PHPhomework1.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$sum = 0;
+for ( $i = 1; $i <= 50 ; $i = $i + 2 ) {
+    $sum = $sum + $i;
+}
+echo "The sum of odd numbers from 0 to 50 = ".$sum."<br>";
```

Comment:
update like below
`echo 'The sum of odd numbers from 0 to 50 = ' . $sum . '<br/>';`
or
`echo "The sum of odd numbers from 0 to 50 = $sum<br/>";`